zuul.d/projects.yaml

```diff
@@ -8,12 +8,9 @@
             dependencies:
               - openstack-k8s-operators-content-provider
         - podified-multinode-edpm-deployment-crc: *content_provider
-        - cifmw-data-plane-adoption-osp-17-to-extracted-crc:
-            dependencies:
-              - openstack-k8s-operators-content-provider
-            files:
-              - ^devsetup/Makefile
-              - ^devsetup/scripts/*
-              - ^devsetup/standalone/*
-              - ^Makefile
-              - ^zuul.d/projects.yaml
```

Comment:
we need this to run the adoption tests

Reply:
Hi @fao89 this is just added for testing, I should have marked the patch as draft :)